code/game/objects/items/devices/laserpointer.dm

```diff
@@ -134,7 +134,7 @@
 
 	//catpeople
 	for(var/mob/living/carbon/human/H in view(1,targloc))
-		if(!isfelinid(H) || H.incapacitated() || H.eye_blind )
+		if( H.incapacitated() || H.eye_blind )

```

Comment:
This makes all humans grab for the laser pointers

Reply:
It makes them what for the laser pointers

Reply:
Good change tbh

Reply:
How do you keep doing this? I fixed this before